app/views/contributions/_contribution_list.erb

```diff
@@ -14,14 +14,14 @@
     <span class="deadmark">
 
     </span>
-    <%= @contribution.content %>
+    <%= @contributions.content %>
```

Comment:
This variable doesn't exist. I'll fix it from my computer tomorrow morning
before merging if nobody solves it before

On Tue, Apr 19, 2016, 00:07 Quim Motger <notifications@github.com> wrote:

> In app/views/contributions/_contribution_list.erb
> <https://github.com/quim-motger/asw-hackernews/pull/4#discussion_r60143023>
> :
>
> > @@ -14,14 +14,14 @@
> >      <span class="deadmark">
> >
> >      </span>
> > -    <%= @contribution.content %>
> > +    <%= @contributions.content %>
>
> Why?? Can I fix it? I don't want to leave it for tomorrow.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quim-motger/asw-hackernews/pull/4/files/f72e40f0a7f979d20bddd6ea3cf37beff0e4a442#r60143023>
>